pulp_2to3_migration/app/plugin/docker/utils.py

```diff
@@ -0,0 +1,21 @@
+from . import pulp2_models
+
+from pulp_2to3_migration.pulp2 import connection
+
+
+connection.initialize()
+
+
+def find_tags():
+    """
+    Find tags that have same name within the repo.
+    Return only one tag out of 2 tags with the same name.
+    Prefer schema2 over schema1.
+    """
+    sort_stage = {'$sort': {'schema_version': -1}}
+    group_stage1 = {'$group': {'_id': {'name': '$name', 'repo_id': '$repo_id'},
+                    'tags_id': {'$first': '$_id'}}}
+    group_stage2 = {'$group': {'_id': None, 'tags_ids': {'$addToSet': '$tags_id'}}}
+    result = pulp2_models.Tag.objects.aggregate([sort_stage, group_stage1, group_stage2])
+
+    return result.next()['tags_ids']
```

Comment:
Have you tested if the result is empty? I'm worried that it will raise a StopIteraiton and we might need to handle that.

Reply:
good catch, fixed